Tests/SymbolKitTests/SymbolGraph/SymbolGraphTests.swift

```diff
@@ -480,3 +502,108 @@ private func encodedLegacySymbolGraph() -> String {
   }
   """
 }
+
+private let obsoleteSymbolOverload: String = """
+{
+  "kind": {
+    "identifier": "swift.init",
+    "displayName": "Initializer"
+  },
+  "identifier": {
+    "precise": "c:objc(cs)UIColor(cm)colorNamed:inBundle:compatibleWithTraitCollection:",
+    "interfaceLanguage": "swift"
+  },
+  "pathComponents": [
+    "UIColor",
+    "init(named:inBundle:compatibleWithTraitCollection:)"
+  ],
+  "names": {
+    "title": "init(named:inBundle:compatibleWithTraitCollection:)",
+  },
+  "declarationFragments" : [
+    {
+      "kind" : "identifier",
+      "spelling" : "init(named:inBundle:compatibleWithTraitCollection:)"
```

Comment:
I don't think we should include real symbol names or identifiers from any real project in the repo as test data, no matter what that project is.

This data is already manipulated to be minimized (the real symbol would have many different declaration fragments of different kinds). I don't see a reason not to also change the `identifier.precise`, `pathComponents`, `names.title`, `declarationFragments[0].spelling`, and `availability[0].renamed` to use made up values (and make the corresponding changes to the other symbol).